temporal/t.rast.accumulate/t.rast.accumulate.py

```diff
@@ -244,6 +244,9 @@ def main():
 
     # The lower threshold space time raster dataset
     if lower:
+        # TODO: Check if range is defined elsewhere than this file.
+        # Otherwise, the code is unreachable as range is a class, built into Python,
+        # used in for loops notably. See t.rast.accdetect for usage of range_
```

Comment:
![image](https://github.com/user-attachments/assets/ef02bc12-93b5-4212-9301-25ad4da557bc)
![image](https://github.com/user-attachments/assets/eb3216b3-346f-4fbe-aa32-03c8f952babe)

We also can see from my fork's PR for code coverage also show that this part of code isn't tested:
https://app.codecov.io/gh/echoix/grass/pull/168/blob/temporal/t.rast.accumulate/t.rast.accumulate.py
![image](https://github.com/user-attachments/assets/a1bfa0f9-b2b6-461b-8a4c-aaf9d359ffa7)

Reply:
After looking into `t.rast.accdetect` and comparing it to the code here, my conclusion is that the codeblock behind `if not range:` is a copy-paste leftover and can be removed. As you say it is never reached and here the value range is defined by upper and lower...

Reply:
So I think it might be best to remove it in another PR

Reply:
See #4154

Reply:
```suggestion
```